src/components/Controls/Controls.tsx

```diff
@@ -35,17 +39,22 @@ const useStyles = makeStyles((theme: Theme) =>
   })
 );
 
-export default function Controls() {
+export default function Controls(props: { disabled?: boolean }) {
   const classes = useStyles();
   const roomState = useRoomState();
   const isReconnecting = roomState === ROOM_STATE.RECONNECTING;
+  const isdisconnected = roomState === ROOM_STATE.DISCONNECTED;
   const isUserActive = useIsUserActive();
+  const [notification, setNotification] = useState(null);
```

Comment:
please try to use the existing notification dialog instead of a local copy of it.